src/Outsystems.SetupTools/Lib/PlatformSetup.ps1

```diff
@@ -696,6 +696,20 @@ function GetServiceStudioInstallDir([string]$MajorVersion)
     return $output
 }
 
+function GetIntegrationStudioInstallDir([string]$MajorVersion)
+{
+    LogMessage -Function $($MyInvocation.Mycommand) -Phase 1 -Stream 2 -Message "Getting the contents of the registry key HKLM:SOFTWARE\OutSystems\Installer\Integration Studio $MajorVersion\(default)"
+    $output = RegRead -Path "HKLM:SOFTWARE\OutSystems\Installer\Integration Studio $MajorVersion" -Name "(default)" #TODO: Add .0 here
```

Comment:
what is this TODO?

Reply:
I will remove it, it was kept because I reused the same function as the Service  Studio